go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions/v3 v3.16.1
 	github.com/giantswarm/app/v4 v4.2.0
-	github.com/giantswarm/appcatalog v0.3.2
+	github.com/giantswarm/appcatalog v0.3.3-0.20210201180833-3fd9d0628bdb
```

Comment:
TODO: update this after release https://github.com/giantswarm/appcatalog/pull/55